R/get_ref.R

```diff
@@ -335,66 +365,78 @@ cond_parse_pkg_ref_release <- function(remote_ref) {
   }
 }
 
+#' Get package version.
+#'
+#' @inheritParams get_ref_min
+#' @returns Package version created with `package_version`.
+#'
+#' @export
+get_version <- function(remote_ref) {
+  UseMethod("get_version", remote_ref)
+}
+
+#' @rdname get_version
+#' @importFrom pkgdepends new_pkg_deps
+#' @exportS3Method get_version remote_ref
```

Comment:
Are you adding this to be explicit? instead of using `#' @export` (that detects it's a S3method and uses the write expression on NAMESPACE)

Reply:
good point - will change it to the basic `#' @export`